### PR TITLE
trunk

### PR DESCRIPTION
**Pull request by DawChihLiou**
Hey everyone👋

[trunk](https://trunk.io) is a new analysis tool that provides linters and formatters for multiple languages. It's designed to run analyses against all the languages in a repository in one go.

You can find the supported linters and formatters in the [official doc](https://docs.trunk.io/check/supported-linters).

Please note that it is on `beta` now. It's for profit(The startup is backed by a16z) and not an open source project.

### CODE REVIEW

**Review by jakubsacha — APPROVED** [review]
Thanks for the contribution. @mre anything against merging it?

**Comment by mre** [comment]
Yeah let's go ahead and merge it. Looks like a stunning tool and I love the dashboard view. Don't know why it was missing from the list for so long. 🤔 😅 
Thanks for adding @DawChihLiou and for reviewing @jakubsacha.

**Comment by mre** [comment]
> Please note that it is on beta now. 

Ah that explains why it was missing: it's a relatively new (and exciting) player.

File: data/tools/trunk.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - ruby
+  - rust
+  - terraform
+license: proprietary
+types:
+  - cli
+source: "https://github.com/trunk-io/"
+homepage: "https://trunk.io"
+description: Modern repositories include many technologies, each with its own set of linters. With 30+ linters and counting, Trunk makes it dead-simple to identify, install, configure, and run the right linters, static analyzers, and formatters for all your repos.
+resources:
```
Comment (mre) [inline data/tools/trunk.yml:27]: Special thanks for adding a `resources` section @DawChihLiou.

**Comment by DawChihLiou** [comment]
Thanks a lot guys @mre @jakubsacha ❤️

**Pull request merged**